Yoast/wpseo-news
# CS/best practices: only one class per file

## Summary

This PR can be summarized in the following changelog entry:
* _N/A_

## Relevant technical choices:
* No functional changes.
* Code style compliance.

Split a plugin file which contained more than one class into separate class files.

## Test instructions

This PR can be tested by following these steps:
* Do a `composer install` to regenerate the autoload file
* Verify that the NewsSEO admin page still displays all fields correctly like before.
